network/wsNetwork.go

```diff
@@ -397,21 +410,34 @@ type WebsocketNetwork struct {
 	// to be sent to new peers.  This is filled in at network start,
 	// at which point messagesOfInterestEncoded is set to prevent
 	// further changes.
-	messagesOfInterestEnc     []byte
-	messagesOfInterestEncoded bool
+	messagesOfInterestEnc        []byte
+	messagesOfInterestEncoded    bool
+	messagesOfInterestGeneration uint32
 
 	// messagesOfInterestMu protects messagesOfInterest and ensures
 	// that messagesOfInterestEnc does not change once it is set during
 	// network start.
-	messagesOfInterestMu deadlock.Mutex
+	messagesOfInterestMu   deadlock.Mutex
+	messagesOfInterestCond *sync.Cond
 
 	// peersConnectivityCheckTicker is the timer for testing that all the connected peers
 	// are still transmitting or receiving information. The channel produced by this ticker
 	// is consumed by any of the messageHandlerThread(s). The ticker itself is created during
 	// Start(), and being shut down when Stop() is called.
 	peersConnectivityCheckTicker *time.Ticker
+
+	node NodeInfo
```

Comment:
could you call this something other than `node`, maybe `nodeInfo`? We already have a bunch of Node types

Reply:
I had it as `NetworkNodeInfo`, being the part of the node info that network cares about, but lint complained 'this stutters' because outside the package it would be `network.NetworkNodeInfo`

Reply:
I meant the name of the field, not the type — elsewhere in the code it was confusing for me to see `wn.node` used and thought maybe it could be called `wn.nodeInfo` instead